src/main/scala/com/thirdship/libunit/CompoundTSUnit.scala

```diff
@@ -165,17 +165,22 @@ class CompoundTSUnit(val numerator: List[TSUnit] = List.empty[TSUnit],
 		}
 	}
 
+	private def keySetEquals(unit: CompoundTSUnit): Boolean = {
+		numeratorDimensions.keySet.equals(unit.numeratorDimensions.keySet) &&
```

Comment:
I noticed, as I was going through the code, `equateUnitLists` checks the units themselves, whereas `keySetEquals` checks to see that they are the same kind of unit, and thus can be converted between them. So there is a subtle difference between them, making it difficult to eliminate one or the other. Maybe I can use one inside the other, to save some computation time and give `keySetEquals` another use?